LICENSE.txt

```diff
@@ -0,0 +1,222 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
@brucelit I added the GNU AFFERO LICENSE because the project includes PM4Py

But PM4Py is only used for passing the Petri net and Event log. So we can consider reworking this if you want to have a more permissive license